### PR TITLE
fixes #2 added authentication

### LINKED ISSUE #2: Add authentication
@abhishek1208 @tech4GT

### PR DESCRIPTION
**Pull request by abhishek1208**
added authentication and acl

### CODE REVIEW

**Comment by abhishek1208** [comment]
@championswimmer Please review!

File: secrets.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+
+  "clientID": "8473906170",
+  "clientSecret": "CwgBS7nr90EFyLF7u3TGPnwRHhhNt4MU57BbgJESEldSPiN2azHHaqvwHPpTVrfB"
```
Comment (championswimmer) [inline secrets.json:4]: should not have committed this, but it;s ok, well generate new keys and use a different app

**Review by championswimmer — APPROVED** [review]

File: secrets.json (hunk 1 above)
Comment (abhishek1208) [reply secrets.json:4]: Will keep that in mind!

**Comment by abhishek1208** [comment]
@championswimmer Sir, is it really required to check the role of the user in acl because we are not allowing login for any user other than the admins?

**Pull request merged**